packages/upswyng-web/src/components/Hotlines.tsx

```diff
@@ -1,9 +1,105 @@
+    phone: "1-877-565-8860",
+  },
+  {
+    name: "Substance Abuse and Mental Health Services Administration",
+    description:
+      "SAMHSA’s National Helpline (also known as the Treatment Referral Routing Service) is a confidential, free, 24-hour-a-day, 365-day-a-year, information service, in English and Spanish, for individuals and family members facing mental and/or substance use disorders. This service provides referrals to local treatment facilities, support groups, and community-based organizations. Callers can also order free publications and other information.",
+    phone: "1-800-662-4357",
+  },
+  {
+    name: "National Hotline for Crime Victims",
+    description:
+      "The VictimConnect Resource Center is a place for crime victims to learn about their rights and options confidentially and compassionately. VictimConnect serves victims of any crime in the United States. Whether by chat or phone, our specialists assist victims in locating appropriate national, local, or online referrals. VictimConnect also has a special focus on populations, crimes, and topics that are generally underrepresented or underserved in victim services.",
+    phone: "1-855-484-2846",
+  },
+  {
+    name: "National Disaster Distress Helpline",
+    description:
+      "The Disaster Distress Helpline is a national hotline dedicated to providing year-round immediate crisis counseling for people who are experiencing emotional distress related to any natural or human-caused disaster. This toll-free, multilingual, and confidential crisis support service is available to all residents in the United States and its territories. Stress, anxiety, and other depression-like symptoms are common reactions after a disaster.",
+    phone: "1-855-484-2846",
+  },
+  {
+    name: "United Way",
+    description:
+      "Every day thousands of people across North America turn to 2-1-1 for information and support—whether financial, domestic, health or disaster-related. 2-1-1 is a free, confidential referral and information helpline and website that connects people from all communities and of all ages to the essential health and human services they need, 24 hours a day, seven days a week.",
+    phone: "2-1-1",
+  },
+];
+
 const Hotlines = () => (
-  <div>
-    <p>Hotlines</p>
-  </div>
+  <Container>
+    <PageBanner color={colors.pink} text={"Hotlines"} />
+    <List>
+      {hotlineList.map(({ name, description, phone }, i) => (
+        <HotlineCard
+          key={i}
```

Comment:
Probably fine in this case since this is static, but [best not to use indexes as keys](https://medium.com/@robinpokorny/index-as-a-key-is-an-anti-pattern-e0349aece318). Name would be a better choice.

Reply:
Oh wow, that is good to know!